pytest.ini

```diff
@@ -1,3 +1,7 @@
 [pytest]
 DJANGO_SETTINGS_MODULE=timed.settings
 addopts = --reuse-db
+env =
+  DJANGO_AUTH_LDAP_ENABLED=True
+  DJANGO_AUTH_LDAP_SERVER_URI=ldap://127.0.0.1
+  DJANGO_AUTH_LDAP_USER_DN_TEMPLATE=uid=%(user)s,ou=people,o=test
```

Comment:
If we're enabling LDAP auth here and then testing that, are we even testing the regular DB auth?

Reply:
Yes... because it enables two backends. First it will always tries ldap backend if user doesn't exist it will try db backend. The same way as it also runs on Timed Production Server.